### ISSUE TITLE
As a developer, I want to know which collection a pollutant_value belongs to

### ISSUE THREAD

**Issue by TangoYankee** [openoakland/woeip#238]
**Is your feature request related to a problem? Please describe.**
`Pollutant_values` does not have a link to the `collection` where it came from. This makes it impossible to display `pollutant_values` based on the collection where they were created.

**Describe the solution you'd like**
I would like to update the `pollutant_value` model to record the id of the `collection` that created it.

**Describe alternatives you've considered**
The alternative is to infer the collection based on the datetime of the reading. However, this will cause erroneous results if two or more collections/sessions overlap. This will also make it difficult to eventually track which pollutant values have been validated.

**Additional context**
The `collection` id will now have to be recorded whenever the pollutant values are saved to the database.

**Labels**
- [x] Add a status label
  - `backlog`
  - `discussing`
  - `ready`
  - `implementing`
- [x] Add additional labels as needed

### For Developers
**Affected Components**
List what pages or backend files will need to be added, removed, or changed

**Technical Resources**
Add links to appropriate documentation. Describe possible code implementations.

**Design Documentation**
Wireframes or other design resources

**Testing Outline**
Describe the testing methods appropriate for this feature

**Comment by theecrit** [comment]
Shouldn't you be able to get to the related Collection via the pollutantValue's `collection_ID`, which has an associated  `collection_file_ID` per the [schema](https://drive.google.com/file/d/1bm2X4h3X-UnezP2JhbB_El41dQ4nUi2X/view?usp=sharing)? 

I'll admit that the schema may be overly complicated but we got there through a *lot* of conversation and iteration in an attempt to make it scalable to other types of devices, sensors, and pollutant types. It may be that we need to re-document the *existing* schema and then revisit the overall model. It also may be that we haven't implemented the existing schema accurately in the first place.

**Comment by TangoYankee** [comment]
@theecrit yes, your interpretation of the schema is correct and this issue is wrong. I updated my original slack message to reflect that I initially wrote this off the Swagger docs, which are incomplete. When I went through the code, I saw the relationship of pollutant_value -> collection_file -> collection. 

As for the fate of this issue, my plan was to ghost delete it by simply changing it to an actual code improvement. That plan seems to be foiled :P
